frontend/src/App.tsx

```diff
@@ -8,39 +8,43 @@ import DataHelpers from './components/DataHelpers';
 import ItemType from './types/ItemType';
 import FilterType from './types/FilterType';
 
+import services from './services.js';
+
+// TODO: is this the right place to declare this constant?
+let BACKEND_URL:string = process.env.REACT_APP_BACKEND_URL || 'localhost:8000';
+// console.log(process.env, BACKEND_URL);
+BACKEND_URL = 'http://' + BACKEND_URL;
```

Comment:
Berrer to make this constant:
```suggestion
const BACKEND_URL = `http://${process.env.REACT_APP_BACKEND_URL || 'localhost:8000'}`;
```